eng/common/TestResources/Remove-TestResources.ps1

```diff
@@ -38,6 +38,9 @@ param (
     [Parameter(ParameterSetName = 'Default+Provisioner', Mandatory = $true)]
     [Parameter(ParameterSetName = 'ResourceGroup+Provisioner', Mandatory = $true)]
     [string] $ProvisionerApplicationSecret,
+    
+    [Parameter(Mandatory = $false)]
+    [string] $ServiceDirectory,
```

Comment:
When adding a parameter here we should make it a parameter in [`remove-test-resources.yml`](https://github.com/Azure/azure-sdk-tools/blob/master/eng/common/TestResources/remove-test-resources.yml#L9)

Something like: 

```yaml
parameters:
  ServiceDirectory: ''
```